force-app/main/default/aura/AccountMap/AccountMapRenderer.js

```diff
@@ -0,0 +1,5 @@
+({
```

Comment:
It is unclear to me why you have created these resources when you aren't using them immediately. Is there a reason you would like to elaborate on?